distro/pom.xml

```diff
@@ -293,19 +291,87 @@
+                  token='source utils.sh'>
+                  <replacevalue>
+                    source utils.sh&#10;&#10;source msf_utils.sh
+                  </replacevalue>
+                </replace>
+
+                <echo level="info" message="Adding MSF environment variables to Ozone" />
+                <replace
+                  file="${project.build.directory}/${project.artifactId}-${project.version}/run/docker/scripts/utils.sh"
+                  token='echo "→ ERPNEXT_SCRIPTS_PATH=$ERPNEXT_SCRIPTS_PATH"'>
+                  <replacevalue>
+                    echo "→ ERPNEXT_SCRIPTS_PATH=$ERPNEXT_SCRIPTS_PATH"&#10;&#9;setMSFEnvironment
+                  </replacevalue>
+                </replace>
+                <replace
+                  file="${project.build.directory}/${project.artifactId}-${project.version}/run/docker/scripts/utils.sh"
+                  token='echo "→ DISTRO_PATH=$DISTRO_PATH"'>
+                  <replacevalue>
+                    echo "→ DISTRO_PATH=$DISTRO_PATH"
+                    export MSF_ENV_CONFIG_PATH=${project.build.directory}/${project.artifactId}-${project.version}/run/docker
+                    export MSF_SERVER_TYPE="${msfServerType}"
+                    echo "→ MSF_SERVER_TYPE=$MSF_SERVER_TYPE"
+                  </replacevalue>
+                </replace>
+                <replace
+                  file="${project.build.directory}/${project.artifactId}-${project.version}/run/docker/scripts/utils.sh"
+                  token='echo "$INFO Exporting Traefik hostnames..."'>
+                  <replacevalue>
+                    echo "$INFO Exporting Traefik hostnames..."
+                    export OPENFN_HOSTNAME=openfn-"${IP_WITH_DASHES}.traefik.me"
+                    echo "→ OPENFN_HOSTNAME=$OPENFN_HOSTNAME"
+                  </replacevalue>
+                </replace>
+
+                <echo level="info" message="Adding traefik to Ozone start script" />
+                <replace
+                  file="${project.build.directory}/${project.artifactId}-${project.version}/run/docker/scripts/start.sh"
+                  token='echo "$INFO Skipping running Nginx proxy... (\$TRAEFIK=true)"'>
+                  <replacevalue>
+                    runMSFServerTraefikAndNginxProxy
+                  </replacevalue>
+                </replace>proxy_pass http://fhir-odoo:8080;
+
+                <!-- Todo: Remove this when fhir-odoo nolonger breaks nginx-proxy -->
+                <echo level="info" message="Fixing nginx proxy not starting" />
+                <replace
+                  file="${project.build.directory}/${project.artifactId}-${project.version}/run/docker/proxy/default.conf.template"
+                  token='proxy_pass http://fhir-odoo:8080;'>
+                  <replacevalue>
+                    set $fhirOdd fhir-odoo:8080;
+                    proxy_pass http://$fhirOdd;
+                  </replacevalue>
+                </replace>
```

Comment:
this fixes nginix proxy failing to start.  here this [comment](https://github.com/ozone-his/ozone-docker-compose/pull/112#discussion_r1802621651)